pkg/apis/psmdb/v1/psmdb_defaults.go

```diff
@@ -170,8 +170,8 @@ func (cr *PerconaServerMongoDB) CheckNSetDefaults(platform version.Platform, log
 			cr.Spec.Sharding.Mongos.Size = 0
 		} else {
 			if !cr.Spec.UnsafeConf && cr.Spec.Sharding.Mongos.Size < minSafeMongosSize {
-				log.Info(fmt.Sprintf("Mongos size will be changed from %d to %d due to safe config", cr.Spec.Sharding.Mongos.Size, minSafeMongosSize))
-				log.Info("Set allowUnsafeConfigurations=true to disable safe configuration")
```

Comment:
Why do we decide to remove this message?

Reply:
If you look at the following log messages we have I think it is unnecessary. And for the second log line, which is obvious from our docs (if it is not it should be), I don't think we should put it in our logs.